develop/integrations.rst

```diff
@@ -1,6 +1,183 @@
 External Integrations
 #####################
 
+.. _cdc:
+
+Change Data Capture (CDC)
+=========================
+
+.. note::
+
+  Logical decoding for distributed tables is a beta feature in Citus v11.3.
```

Comment:
beta -> Preview
In the release updates, we are refering to it as 'Preview' release